core/src/main/kotlin/com/gatehill/corebot/action/BaseJobTriggerService.kt

```diff
@@ -73,6 +73,9 @@ abstract class BaseJobTriggerService(private val lockService: LockService,
         } else {
             // already finished
             actionOutcomeService.handleFinalStatus(trigger, action, executionDetails.id, executionDetails.status)
+            if (action.options.get("show-job-output")?.value == "true") {
```

Comment:
So, the `options` block represents values sent to the backend (like Jenkins or Rundeck) when executing the job - in Jenkins they're build parameters and in Rundeck they're called options, but it's the same concept.

What you are looking for here is an update to the action config model; the ability to turn on/off showing output is a configuration setting for a job, rather than something you want to be passed to the backend.

Would you be able to move it into the action config model?